service/matching/db.go

```diff
@@ -33,6 +33,7 @@ import (
 	"github.com/temporalio/temporal/common/log/tag"
 	"github.com/temporalio/temporal/common/persistence"
 	"github.com/temporalio/temporal/common/primitives"
+	tasklistpb "go.temporal.io/temporal-proto/tasklist"
```

Comment:
I think you need to run `make goimports` to fix ordering here.

Reply:
hmm... my editor does this automatically, and then i ran `make goimports` by hand, just to make sure, and the order didn't change. are you expecting something different here?

Reply:
There should be 3 blocks:
1. system imports
2. third party libs
3. internal packages

`go.temporal.io/temporal-proto/` considered to be 3rd party in this case.

![image](https://user-images.githubusercontent.com/2232524/80764489-a8183200-8af5-11ea-806b-1262017e3e2c.png)

Reply:
It seems grouping is GoLand feature not goimports.

Reply:
You can ignore this then.